tests/installation/select_patterns_and_packages.pm

```diff
@@ -28,7 +28,10 @@ use testapi;
 sub accept3rdparty {
     #Third party licenses sometimes appear
     while (check_screen([qw(3rdpartylicense automatic-changes inst-overview)], 15)) {
-        last if match_has_tag("automatic-changes");
+        if (match_has_tag("automatic-changes")) {
+            send_key 'alt-o';
+            last;
+        }
```

Comment:
Please do not just change this within the very same commit. This demands way more verbose explanation and good reasoning why this should be done. And much more testing than just a single test job as so many other test scenarios are potentially involved.

Reply:
Yes, this is a regression, might be, I will separate it into another PR.